etna/ciim/constants.py

```diff
@@ -36,6 +53,27 @@ class Bucket:
     is_current: bool = False
     results: List[Any] = None
 
+    # Fetch more 'groups' so that we receive counts for any bucket/tab options we might be showing Ex: ["group:30"]
+    aggregations: List[str] = field(default_factory=list)
+
+    @property
+    def aggregations_normalised(self) -> List[str]:
+        """
+        Returns a list of strings to include as the 'aggregations' option value when assembling
+        an API request for this bucket. Each string must be in the format "filter-name:item-count".
+        """
+        values = []
+        for aggregation in self.aggregations:
+            bits = aggregation.split(":")
+            if len(bits) == 2:
+                values.append(bits[0] + ":" + bits[1])
+            else:
+                values.append(bits[0] + ":10")
+        return values
+
+    def __post_init__(self):
+        self.aggregations = self.aggregations_normalised
+
```

Comment:
Hmm. My feedback on this seems to have been removed. Could we remove this please, and just use 'bucket. aggregations_normalised' in the view?

Reply:
> Hmm. My feedback on this seems to have been gone missing.
Sorry I did not receive email notification either. However there is one above for `self.aggregations_normalised`

Reply:
I've now updated this method in the commit 
> Updated for default aggregations and comments [e174551c](https://github.com/nationalarchives/ds-wagtail/pull/808/commits/e174551cb7db57d32fa5e682625de2911f931675)